verticaltabs.js

```diff
@@ -698,10 +697,12 @@ VerticalTabs.prototype = {
     };
     window.addEventListener('aftercustomization', afterListener);
 
-    window.addEventListener('resize', () => {
+    function resizeListener() {
       this.resizeTabs();
       document.documentElement.style.setProperty('--pinned-width', `${Math.min(this.pinnedWidth, document.width / 2)}px`);
-    }, false);
+    }
+    let boundResizeListener = resizeListener.bind(this);
```

Comment:
Let's do the same fat-arrow function trick here that we did for `checkBrighttext` above…